dataset/tags.json

```
@@ -269,6 +269,16 @@
       "zh-CN": "原神中的每日委托的英语和日语翻译"
     }
   },
+  "quest-tribal": {
+    "en": "Tribal Chronicles",
```

I want to make English tag names singular (単数形) because the tags are added to each single word.
For example, this tag is added to "Yupanqui's Turnfire", and this quest is **not** a group of Tribal Chronicle**s**, but a single Tribal Chronicle.

```suggestion
    "en": "Tribal Chronicle",
```